api/README.md

```diff
@@ -271,24 +220,14 @@ spec:
             memory: "150Mi"
 ```
 
-The `namespace` of the resource mentioned in `.spec.data` will be defaulted to `.spec.sync.namespace` or `.metadata.namespace` of the referencing `Kyma` if not filled, otherwise it will be respected. All other attributes (including `.metadata.name` and `apiVersion` and `kind`) are taken over as is. Note that since it behaves similarly to a `template` any subresources, like `status` are ignored, even if specified in the field.
+The `namespace` of the resource mentioned in `.spec.data` will be defaulted to `.metadata.namespace` of the referencing `Kyma` if not filled, otherwise it will be respected. All other attributes (including `.metadata.name` and `apiVersion` and `kind`) are taken over as is. Note that since it behaves similarly to a `template` any subresources, like `status` are ignored, even if specified in the field.
```

Comment:
This is not the current implementation, the namespace will be controlled by flag `sync-namespace` if  `.spec.data` not mentioned